sg/datatablesbundle/1.0/config/routes/sg_datatables_bundle.yaml

```diff
@@ -0,0 +1,3 @@
+sg_datatables_bundle:
+    resource: "@SgDatatablesBundle/Controller/"
+    type:     annotation
```

Comment:
I haven't had a look at the bundle, but don't we need a `prefix` here?

Reply:
There wasn't a global bundle prefix, I just added one moving it from the controller annotation on https://github.com/stwe/DatatablesBundle/pull/716. I updated their documentation and now I'll proceed to update the flex recipe also. I think it's a good practice to have a global bundle routing prefix.